tac/guidelines/project-consistency-guidelines.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Project Consistency
+parent: Guidelines
+grand_parent: LF Decentralized Trust TAC
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+This document outlines consistent approaches to certain aspects of running and maintaining an LFDT project.
+
+LFDT projects will always have their own approaches to how they manage and govern themselves, how they
+communicate and collaborate, and how they track and communicate progress.
+
+The LFDT already sets out `MUST` and `SHOULD` and `MAY` behaviours that projects should treat accordingly. This document
+is not intended to replace any of the existing governance rules set out by the LFDT. Instead it is designed improve
+consistency across projects such that inter-project collaboration is easier, and to make it easier for individuals to
+collaborate to multiple LFDT projects.
+
+**MRW** - There is potentially a lot of overlap with [project best practices](./project-best-practices.md)
+
+**MRW** - Perhaps this document should be wrapped into that document?
+
+## Relationship to existing guidelines
+
+ - MRW: Suggest some discussion around consistency vs guidelines
+
+### Areas of consistency by stakeholder
+
+1. LFDT
+   - Communication
+      - Release activity
+   - CVE bounties
+3. Project users
+   - Communication
+      - **Readme structure that gives me a quick overview of what the project is/does** 
+      - Release activity
+      - **CVE resolution should be included in release notes including specific CVE numbers**
+         - (Potentially GH built-in CVE labelling)
+      - CVE advisories
+   - Issue tracking
+      - Progress on issues & bug fixes
+   - Roadmap
+      - Where to find it
+      - How to contribute to it (raise issues? attend contributor calls?)
+5. Project maintainers & contributors
+   - Communication
+      - **Readme structure that gives me a quick way to learn how to set up a development environment**
+      - Release planning
+         - **Public project tracking board**
+            - **Github projects for tracking**
+            - Alternatives acceptable, as long as public and not requiring authentication to view
+         - **Project tracking board linked from repo landing page**
+   - CVE reporting
+      - **Raising GH advisories for at least HIGH and CRITICAL CVEs??**
+   - Bug tracking
+   - Roadmap
+      - Where to find it 
+
+
+---
+**OLD NOTES BELOW**
+---
+
+### Communication
+
+### Bug tracking
+
+ - MRW: There is already guidance about using Github practices such as "bug: ..." issue naming
+
+### Release planning
+
+ - MRW: Post on the "maintainers" or "contributors" discord channel that a new release is planned with an approximate timeline
+   - MRW: Is this being too prescriptive?
```

Comment:
@matthew1001 I'd frame it as a suggested checklist intended to be helpful to those who do not have an established process already that they are happy with Lots of new maintainers come in and ask for exactly this kind of advice but there are also people who strongly prefer doing it their own way. I think it's good to have this kind of recommendations as long as it is clear and obvious at first glance that it is a recommendation for those in need some advice and that for others it can be safely disregarded as they see fit.